package.json

```
@@ -41,6 +43,7 @@
         "prepare": "yarn build",
         "xbasic": "yarn --cwd examples/basic",
         "docs": "yarn --cwd docs build",
+        "release": "release-it",
         "test": "echo no test available"
     },
     "files": [
```

we should make sure that next release won't have docs and example 😅
```suggestion
    "files": [
        ...
        "!docs",
        "!examples",
        ...
    ]
```